c10/core/Backend.h

```diff
@@ -31,15 +31,18 @@ enum class Backend {
   CUDA,
   HIP,
   FPGA,
+  XPU,
   SparseCPU,
   SparseCUDA,
   SparseHIP,
+  SparseXPU,
   MSNPU,
   XLA,
   Vulkan,
   Metal,
   QuantizedCPU,
   QuantizedCUDA,
+  QuantizedXPU,
```

Comment:
Is QuantizedXPU actually going to be a thing?

Reply:
Yes. We are already using QuantizedXPU to implement the low precise in XPU.